docs/sdk/src/guides/your_first_pallet/mod.rs

```diff
@@ -319,11 +319,11 @@
 //! - Learn more about the individual pallet items/macros, such as event and errors and call, in
 //!   [`frame::pallet_macros`].
 //!
-//! [`pallet::storage`]: ../../../frame_support/pallet_macros/attr.config.html
+//! [`pallet::storage`]: ../../../frame_support/pallet_macros/attr.storage.html
 //! [`pallet::call`]: ../../../frame_support/pallet_macros/attr.call.html
 //! [`pallet::event`]: ../../../frame_support/pallet_macros/attr.event.html
```

Comment:
In here, I think the proper way would be to link to `frame_support::pallet_macros::event`, instead of using `../../../`.

However the link then resolves a little differently (straight to `frame-support-procedural`) instead of `frame-support`, so in this case `../../../` type of linking is preferable.

Reply:
This should be fixed with https://github.com/paritytech/polkadot-sdk/pull/5023, no?

Reply:
I see, so to recap, with the PR above, `frame_support_procedural` is the right place for the docs, but in that PR @gupnik didn't move all of them, and instead resided to a follow-up, but he is on holidays now :) 

TLDR: 

1. if you jump to `frame_support_procedural`, that's fine
2. in the meantime, you can try and follow in nikhil's footsptes and perhaps move a few more macros over? :) up to you

Reply:
Ok, understood.

So I switched to the proper links which jump to `frame_support_procedural`, which eventually will contain the docs.

I'll see if I can help with `2.` (separately - so the PR is ready for review).